aws/services/iam/resource.ftl

```diff
@@ -105,6 +106,8 @@
         ]
     [/#list]
 
+    [#local tags=getOccurrenceCoreTags(occurrence, name) ]
+
```

Comment:
The occurrence isn't passed in as a parameter here. How would the occurrence be found?

Reply:
You're right. I'm now very confused as to why no error, and where the value came from. That's at odds with the freemarker doco on local variables (incl parameters) unless I'm missing something

Reply:
Updated to pass tags and handle legacy account not passing anything to use getCfTemplateCoreTags(name)